### PR TITLE
Move content out of advanced guides into Universal Profiles Learn section

### PR DESCRIPTION
**Pull request by CJ42**
- Remove duplicated guides kept by mistake (Retrieve Token Type, Deploy Token with Metadata...)
- Remove `assets/` folder kept by mistake which showed under the guides in the sidebar ❌
- Moved the expert-guides/universal-profile as a submenu under the `learn/universal-profile` section

image

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -134,15 +134,15 @@ export default {
           // Learn: Working with Assets Section
           {
             from: '/learn/smart-contract-developerss/retrieve-token-type',
-            to: '/learn/assets/retrieve-token-type',
+            to: '/learn/digital-assets/retrieve-token-type',
```
Comment (Hugoo) [inline docusaurus.config.js:137]: I see the line above has 2 s:

```
smart-contract-developerss
```

weird - i think one is enough. I guess the current redirect is not good.

Comment (CJ42) [reply docusaurus.config.js:137]: good catch!

File: docusaurus.config.js [hunk 2]
```diff
@@ -180,11 +180,11 @@ export default {
           },
           {
             from: '/learn/expert-guides/universal-profile/deploy-universal-profile',
-            to: '/learn/expert-guides/deploy-up-with-lsp23',
+            to: '/learn/universal-profile/advanced-guides/deploy-up-with-lsp23',
           },
           {
             from: '/learn/expert-guides/universal-profile/create-profile',
-            to: '/learn/expert-guides/deploy-up-with-lsp23',
+            to: '/learn/universal-profile/advanced-guides/deploy-up-with-lsp23',
```
Comment (Hugoo) [inline docusaurus.config.js:187]: I suggest to add another redirect:

```
from: '/learn/expert-guides/deploy-up-with-lsp23',
to: '/learn/universal-profile/advanced-guides/deploy-up-with-lsp23'
```

**Review by Hugoo — APPROVED** [review]
if you can add these 2 changesw in the redirects then good to merge ;)

nice PR!

**Pull request merged**